### ISSUE TITLE
Benchmarking

### ISSUE THREAD

**Issue by Ishaan-Datta** [UBCAgroBot/ROS#7]
Metric for accuracy in display node (find sample data)
- [ ]  Create performance report for ONNX to TensorRT for both Pytorch and Tensorflow converted models respectively
- [ ]  Create performance report for Pytortch to Torch-TRT
- [ ]  Create performance report for Tensorflow to TF-TRT

Profile ROS2 Nodes:
- [ ]  ros2 topic hz Command: measure the publication rate and latency of a topic
- [ ]  ros2 latency Package: measuring round-trip latency between nodes
- [ ]  ros2 tracing: Use rqt with the ros2_trace plugin for visualization
- [ ]  rclcpp Profiler: ROS2 provides a built-in profiler for C++ nodes
- [ ]  System Profilers (e.g., perf, gprof, valgrind): Example with perf: sudo perf record -e cycles -g -- ros2 run your_package your_node
